oss-cloud-backend/serverless.yml

```diff
@@ -0,0 +1,118 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: oss-cloud-backend
+app: myapp
```

Comment:
Are "app" and "org" properties added by default when Serverless project is initialized? Cannot find them in Serverless docs...
Anyhow, we should change these values.

Reply:
Serverless CLI requested me to enter them, so I've put these placeholders. They will be changed later when we deploy the application online.

Reply:
Line 2 and line 3 are now removed,  because @bandjeo had successfully run the server without these placeholders.